### PR TITLE
fix(clerk-js): Auto focus on password field after email input

### PR DESCRIPTION
**Pull request by desiprisg**
## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

### CODE REVIEW

File: .changeset/stale-starfishes-watch.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': minor
+---
+
+The password field will now autofocus on the factor one page.
```
Comment (gkats) [inline .changeset/stale-starfishes-watch.md:5]:
```suggestion
The password field will now autofocus on the sign in factor one page.
```

**Review by gkats — APPROVED** [review]
:rocket:

**Review by anagstef — APPROVED** [review]

File: packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx [hunk 2]
```diff
@@ -245,7 +245,7 @@ export function _SignInStart(): JSX.Element {
   }
 
   const hasSocialOrWeb3Buttons = !!authenticatableSocialStrategies.length || !!web3FirstFactors.length;
-  const shouldAutofocus = !isMobileDevice() && hasSocialOrWeb3Buttons;
+  const shouldAutofocus = !isMobileDevice() && !hasSocialOrWeb3Buttons;
```
Comment (dimkl) [inline packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx:248]: ❓ could we add a test about this condition? It seems that if there are social or web3 buttons we shouldn't autofocus on email input. right?

**Review by dimkl — APPROVED** [review]

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**